change/@fluentui-react-c04deae4-a13e-45a0-8a98-b7c3f4ca4a5b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
The change was was 'minor' in #26931 -- should this also be 'minor'?
https://github.com/microsoft/fluentui/pull/26931/files#diff-57e25edb4a3e974d4a4f48a6814a05ca73763c9b6e057734fdaf5479b8d7d33e